js/PaymentRequest/PaymentRequestUpdateEvent.js

```diff
@@ -36,10 +38,11 @@ export default class PaymentRequestUpdateEvent {
   constructor(name, target) {
     if (
       name !== SHIPPING_ADDRESS_CHANGE_EVENT &&
-      name !== SHIPPING_OPTION_CHANGE_EVENT
+      name !== SHIPPING_OPTION_CHANGE_EVENT &&
+      name !== PAYMENT_METHOD_CHANGE_EVENT
     ) {
       throw new Error(
-        `Only "${SHIPPING_ADDRESS_CHANGE_EVENT}" and "${SHIPPING_OPTION_CHANGE_EVENT}" event listeners are supported.`
+        `Only "${SHIPPING_ADDRESS_CHANGE_EVENT}" and "${SHIPPING_OPTION_CHANGE_EVENT} and "${PAYMENT_METHOD_CHANGE_EVENT}"" event listeners are supported.`
```

Comment:
```suggestion
        `Only "${SHIPPING_ADDRESS_CHANGE_EVENT}", "${SHIPPING_OPTION_CHANGE_EVENT} and "${PAYMENT_METHOD_CHANGE_EVENT}"" event listeners are supported.`
```

Reply:
(This is a nitpick suggestion just to improve readability.  This change is not essential.)